### DISCUSSION TITLE
运行过一段时间就会报错

### DISCUSSION THREAD

**Discussion by YumengZzzz** [mikumifa/biliTickerBuy#330] [提问专区]
如图
![屏幕截图 2024-07-01 134507](https://github.com/mikumifa/biliTickerBuy/assets/174300125/066f2f21-c997-4b77-81b6-e81b4de3998d)

**Comment by mikumifa** [comment]
Discussion有类似问题，好像是中文路径问题

**Reply by YumengZzzz** [reply]
ok我看看

**Reply by YumengZzzz** [reply]
emm改了英文路径还是报错
![屏幕截图 2024-07-01 141500](https://github.com/mikumifa/biliTickerBuy/assets/174300125/4c275a59-297d-4a68-b1b5-4001a560681a)
然后biliTickerBuy-main这个是啥

**Comment by mikumifa** [comment]
这个问题我还真不太清楚，网上搜一下通用解决方式？

**Reply by YumengZzzz** [reply]
网上有
https://blog.csdn.net/lyb06/article/details/129675526
好像要改程序

**Comment by YumengZzzz** [comment]
删除重新解压是目前有效办法，就是时效性不强
